### PR TITLE
[bitnami/redis-cluster] add info level to logs Signed-off-by: Schneidenwind Christine (LDC) <…

### PR DESCRIPTION
**Pull request by Christine-Schneidenwind**
## Name and Version
bitnami/redis-cluster:7.2.1-debian-11-r0

## What is the problem this feature will solve?
We are collecting our logs via open telemetry. I have seen that there are log entries without severity.
e.g.

```
redis-cluster 09:09:59.12 Subscribe to project updates by watching https://github.com/bitnami/containers
redis-cluster 09:09:59.12 Submit issues and feature requests at https://github.com/bitnami/containers/issues 
Changing old IP x.x.x.x by the new one x.x.x.x 
```

see also Issue: https://github.com/bitnami/containers/issues/51496

## Changing to 
```
redis-cluster 09:09:59.12 INFO Subscribe to project updates by watching https://github.com/bitnami/containers
redis-cluster 09:09:59.12 INFO Submit issues and feature requests at https://github.com/bitnami/containers/issues
09:09:59.12 INFO Changing old IP x.x.x.x by the new one x.x.x.x
```

### CODE REVIEW

File: bitnami/redis-cluster/7.2/debian-11/prebuildfs/opt/bitnami/scripts/libbitnami.sh [hunk 1]
```diff
@@ -45,9 +45,9 @@ print_image_welcome_page() {
     local github_url="https://github.com/bitnami/containers"
 
     log ""
```
Comment (fevisera) [inline bitnami/redis-cluster/7.2/debian-11/prebuildfs/opt/bitnami/scripts/libbitnami.sh:47]: Are there any cons to doing this?
```suggestion
    info ""
```

File: bitnami/redis-cluster/7.2/debian-11/prebuildfs/opt/bitnami/scripts/libbitnami.sh [hunk 2]
```diff
@@ -45,9 +45,9 @@ print_image_welcome_page() {
...
-    log "${BOLD}Welcome to the Bitnami ${BITNAMI_APP_NAME} container${RESET}"
-    log "Subscribe to project updates by watching ${BOLD}${github_url}${RESET}"
-    log "Submit issues and feature requests at ${BOLD}${github_url}/issues${RESET}"
+    info "${BOLD}Welcome to the Bitnami ${BITNAMI_APP_NAME} container${RESET}"
+    info "Subscribe to project updates by watching ${BOLD}${github_url}${RESET}"
+    info "Submit issues and feature requests at ${BOLD}${github_url}/issues${RESET}"
     log ""
```
Comment (fevisera) [inline bitnami/redis-cluster/7.2/debian-11/prebuildfs/opt/bitnami/scripts/libbitnami.sh:51]: Here as well?
```suggestion
    info ""
```

**Comment by fevisera** [comment]
Hi @Christine-Schneidenwind,

Thank you so much for creating this PR. Kindly review the comments I added.

Thanks!

File: bitnami/redis-cluster/7.2/debian-11/prebuildfs/opt/bitnami/scripts/libbitnami.sh (hunk 1 above)
Comment (Christine-Schneidenwind) [reply bitnami/redis-cluster/7.2/debian-11/prebuildfs/opt/bitnami/scripts/libbitnami.sh:47]: Than its not an empty line. Its a line with the log severity. That was the reason I didn't change it. 
But is doesn't metter. So I can do that

**Comment by fevisera** [comment]
Thank you for applying the changes!

**Review by fevisera — APPROVED** [review]
LGTM!

**Pull request merged**